### ISSUE TITLE
ImportError when opening xas_viewer

### ISSUE THREAD

**Issue by jsevarts** [xraypy/xraylarch#316]
@newville Hi, Thanks for putting so much effort into the Larch Project.  I am a bit of a rookie in github and just starting to get my feet wet with x-ray absorption data analysis.  I recently attempted to install xraylarch from the binary installer.  When I attempt to open the xas_viewer.exe, I get the following error (lmfit 1.0.2 installed.)

"ImportError: cannot import name 'CORREL_HEAD' from 'lmfit.printfuncs' (/Users/jonathanevarts/xraylarch/lib/python3.8/site-packages/lmfit/printfuncs.py)"

I saw the error pop up on a couple of old threads so I tried to force install the lmfit 0.9.11.  Now I get another ImportError

"ImportError: cannot import name 'doniach' from 'lmfit.lineshapes' (/Users/jonathanevarts/xraylarch/lib/python3.8/site-packages/lmfit/lineshapes.py)"

Any assistance would be greatly appreciated.  Looking forward to using Larch!

On a side note, the binary installer did not install any desktop shortcuts.  I tried larch -m which gave the same errors above.

I am running MacOS 11.4 on a MacBook Pro 16,1.

Thanks,
Jonathan Evarts

**Comment by laumiulun** [comment]
We are encounter similar issue in Window as well.

Andy

**Comment by laumiulun** [comment]
I applied for a PR with the changes.

https://github.com/lmfit/lmfit-py/pull/760

**Comment by newville** [comment]
@laumiulun @jsevarts Very sorry for the delay and thanks.  Yes, I looked at this and is all my doing.  And thanks for the PR.  I think I'll have to release a new version for this, and hopefully soon!

**Comment by jsevarts** [comment]
@newville @laumiulun Thanks for all the support on the above issue!.  I deleted my previous post because it was such a weird error, I thought a fresh install might fix it.  I reinstalled xraylarch in a conda environment using pip install xraylarch.

However, I ran into the following error when attempting to run xas_viewer. No errors during installation.

"ImportError: cannot import name 'LarchWxApp' from 'larch.wxlib' (/Users/jonathanevarts/opt/anaconda3/envs/xas/lib/python3.9/site-packages/larch/wxlib/__init__.py)"

I saw this error, or similar, on a couple of older threads which were fixed by installing wxpython. so I tried installing wxpython using pip in my conda environment.  Which produced a new error.

 "File "/Users/jonathanevarts/opt/anaconda3/envs/xas/lib/python3.9/site-packages/larch/wxlib/larchframe.py", line 13, in <module>
    from wxutils import (MenuItem, Font, Button, Choice)
ModuleNotFoundError: No module named 'wxutils'"

I ended up installing wxutils and wxmplot, which then lead me to the original second error I got above.

"ImportError: cannot import name 'doniach' from 'lmfit.lineshapes' (/Users/jonathanevarts/xraylarch/lib/python3.8/site-packages/lmfit/lineshapes.py)"

@laumiulun were you also running into this error.  If so, did you have a workaround or perm fixes?

Thanks again!
